chainweb-api.cabal

```diff
@@ -28,7 +28,10 @@ library
     , bytestring            ^>=0.10
     , containers            ^>=0.6
     , hashable              >=1.2 && < 1.4
+    , lens >= 4.16
```

Comment:
It looks like all three of these new deps are unused?  If so, let's remove them.